.github/workflows.yml

```diff
@@ -0,0 +1,32 @@
+on:
+  push:
+    branches:
+    - main
+    - '**'
+  pull_request:
+    branches: 
+    - main
+  workflow_dispatch:
+
+jobs:
+    lint:
+      name: "Code linting and testing"
+      runs-on: ubuntu-latest
+      steps:
+      
+        - name: "Clone repositry"
+          uses: actions/checkout@v2
+        
+        - name: "Python-Setup"
+          uses: actions/setup-python@v5
+          with:
+            python-version: '3.11.7'
+          
+        - name: "Install pylint/pytest"
+          run: pip3 install -r pipeline/requirements.txt
```

Comment:
How does this work if we are in the dashboard folder, I think they will have different requirements.